packages/lib/src/core/Analytics/analyticsPreProcessor.ts

```diff
@@ -59,7 +59,8 @@ export const analyticsPreProcessor = (analyticsModule: AnalyticsModule) => {
                     ...(brand && { brand }),
                     ...(typeof isExpress === 'boolean' && { isExpress }),
                     ...(isExpress === true && hasExpressPage && { expressPage }), // We only care about the expressPage value if isExpress is true
-                    ...(configData && { configData })
+                    ...(configData && { configData }),
+                    ...(originalConfigData && { configData: originalConfigData })
```

Comment:
You should add a comment here to explain what "originalConfigData" represents i.e. config data extracted from the Dropin - since it is not clear form the code itself

Reply:
Sure.